client/src/components/Admin/Jobs/jobComponent.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+    <b-modal size="xl" title="Create Modal" v-model="createJobModal" centered @ok="handleCreate" @cancel="cancelCreateModal"
```

Comment:
wrong title, how will it handle update Job case?
also pls make it job instead of modal